backend/api/migrations/0001_initial.py

```diff
@@ -0,0 +1,118 @@
+                ('is_superuser', models.BooleanField(default=False, help_text='Designates that this user has all permissions without explicitly assigning them.', verbose_name='superuser status')),
+                ('username', models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username')),
+                ('first_name', models.CharField(blank=True, max_length=150, verbose_name='first name')),
+                ('last_name', models.CharField(blank=True, max_length=150, verbose_name='last name')),
+                ('email', models.EmailField(blank=True, max_length=254, verbose_name='email address')),
+                ('is_staff', models.BooleanField(default=False, help_text='Designates whether the user can log into this admin site.', verbose_name='staff status')),
+                ('is_active', models.BooleanField(default=True, help_text='Designates whether this user should be treated as active. Unselect this instead of deleting accounts.', verbose_name='active')),
+                ('date_joined', models.DateTimeField(default=django.utils.timezone.now, verbose_name='date joined')),
+                ('user_id', models.AutoField(primary_key=True, serialize=False)),
+                ('age', models.IntegerField(blank=True, null=True)),
+                ('phone_number', models.TextField(blank=True, null=True)),
+                ('birthday', models.DateField()),
+                ('blood_type', models.CharField(blank=True, max_length=2, null=True)),
+                ('height', models.IntegerField(blank=True, null=True)),
+                ('weight', models.IntegerField(blank=True, null=True)),
+                ('groups', models.ManyToManyField(blank=True, help_text='The groups this user belongs to. A user will get all permissions granted to each of their groups.', related_name='user_set', related_query_name='user', to='auth.group', verbose_name='groups')),
+                ('user_permissions', models.ManyToManyField(blank=True, help_text='Specific permissions for this user.', related_name='user_set', related_query_name='user', to='auth.permission', verbose_name='user permissions')),
+            ],
+            options={
+                'db_table': 'users',
+                'managed': True,
+            },
+            managers=[
+                ('objects', django.contrib.auth.models.UserManager()),
+            ],
+        ),
+        migrations.CreateModel(
+            name='MedicationInfo',
+            fields=[
+                ('medication_id', models.AutoField(primary_key=True, serialize=False)),
+                ('dosage', models.CharField(max_length=8)),
+                ('medication_time', models.JSONField()),
+                ('misc_information', models.CharField(blank=True, max_length=200, null=True)),
```

Comment:
Is this the "notes" that the user can type in? In that case should it have a max length?

Reply:
This would be notes per medication, the 200 max length is arbitrary and can be changed depending on group discussion, we could also just limit length as a check before any additions are made to the table